video/out/hwdec/hwdec_vaapi.c

```diff
@@ -312,6 +326,7 @@ static void try_format_pixfmt(struct ra_hwdec *hw, enum AVPixelFormat pixfmt)
     if (!s || !mp_image_params_valid(&s->params))
         goto err;
     if (try_format_map(hw, s)) {
+        MP_DBG(hw, "Format supported\n");
```

Comment:
Maybe just a bool to true and print a single time in the `err` path whether the format was supported or not. Otherwise this doubles up the amount of log spam for no particular reason.

Reply:
I could combine this with the "Trying format: %s\n" above, but that means you're printing the line _after_ any errors that occur, which is confusing, and the main reason why I did it this way. Would you still prefer that?

Reply:
I think that's fine?

```
[debug] Format supported: rgbaxyz
[debug] Format supported: p1080
[error] foo bar is not a valid bat baz
[error] Format unsupported: yuv420p69
[debug] Format supported: rgba128
```

Doesn't strike me as particularly confusing. It's actually pretty usual for the error messages from downstream API calls to come *before* the high-level error. I mean that's basically how most errors in mpv/libplacebo/etc. work, right?

Reply:
Good enough for me.